.github/workflows/publish-mempool-benchmarks.yml

```diff
@@ -87,15 +95,17 @@ jobs:
           ref: master
           path: ouroboros-network
 
-      - name: Cabal update
+      - name: Create a cabal.project.local file
         run: |
-          cd ouroboros-network
-          cabal update
+          # We need this option to make cardano-crypto-praos use the libsodium
```

Comment:
Not exactly, we need it to tell cabal to include some C files in cardano-crypto-praos that allows us to work with an unmodified libsodium because the modified bits are provided by those c files.

Now that I worded it maybe I'm wrong, and it is precisely what you described, but in any case I'm leaving this here for your consideration 😄